pulseapi/profiles/models.py

```diff
@@ -78,6 +78,12 @@ class UserProfileQuerySet(models.query.QuerySet):
     A queryset for profiles with convenience queries
     """
 
+    def limit(self, value):
+        """
+        Return only the last [limit] number of entries created.
+        """
+        return self.order_by('-id')[:value]
```

Comment:
why do you need `order_by` here? The profile insertion order doesn't really have any significance.